README.md

```diff
@@ -32,6 +32,11 @@ Interact with your containers directly: pause, restart, and stop containers. Lau
 
 Add custom details or interactions for your hosts, containers, and/or processes by creating Scope plugins. Or, just choose from some that others have already written at the GitHub [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
 
+## Who is using Flux in production
```

Comment:
I think it should be `Scope` instead of `Flux`.

Reply:
Fixed.